src/DotNetProjectFile.Analyzers/NuGet/PackageCache.cs

```diff
@@ -174,4 +176,18 @@ bool HasDllFiles(string subDir)
             return null;
         }
     }
+
+    private static NuSpecFile? TryLoadNuSpecFile(IODirectory directory)
+    {
+        if(directory.Files("*.nuspec")?.FirstOrDefault() is { HasValue: true} nuspec)
```

Comment:
I think it's better to try to see if there is a nuspec file with the exact lower-case package name (in case for some reason someone included a second nuspec file) before grabbing a random one. Also I don't think `HasValue: true` is necessary.

Reply:
I agree. I've update it.